python/bmeg/rna_pb2.py

```diff
@@ -84,7 +84,7 @@
     _descriptor.FieldDescriptor(
       name='value', full_name='bmeg.GeneExpression.ExpressionsEntry.value', index=1,
       number=2, type=1, cpp_type=5, label=1,
-      has_default_value=False, default_value=0,
+      has_default_value=False, default_value=float(0),
```

Comment:
Kinda strange all these changed. Any idea why?

Reply:
hmm .

updated to 
```
$ protoc --version
libprotoc 3.5.1
```

The ` default_value=float(0),` is still there, but now there are the following...



```
 _EXPRESSIONSCALE = _descriptor.EnumDescriptor(
   name='ExpressionScale',
@@ -80,14 +79,14 @@ _GENEEXPRESSION_EXPRESSIONSENTRY = _descriptor.Descriptor(
       has_default_value=False, default_value=_b("").decode('utf-8'),
       message_type=None, enum_type=None, containing_type=None,
       is_extension=False, extension_scope=None,
-      options=None),
+      options=None, file=DESCRIPTOR),
     _descriptor.FieldDescriptor(
       name='value', full_name='bmeg.GeneExpression.ExpressionsEntry.value', index=1,
       number=2, type=1, cpp_type=5, label=1,
       has_default_value=False, default_value=float(0),
       message_type=None, enum_type=None, containing_type=None,
       is_extension=False, extension_scope=None,
-      options=None),
+      options=None, file=DESCRIPTOR),
   ],


........

```